docs/_docs/03-ug-mutable-dataset.md

```diff
@@ -90,4 +97,135 @@ hs.refreshIndex("empIndex", "incremental")
 TODO
 
 ### Hybrid Scan
-TODO
+
+Hybrid Scan enables to utilize existing index data along with newly appended source files or
+deleted existing files, without explicit refresh operation. For an index with appended source data files,
+HybridScan changes the query plan to shuffle new data on-the-fly and merge it with index records.
+For an index with deleted source data files, Hyperspace also modifies the plan to exclude the rows from deleted files
+in the index data and for this, you need to set the lineage column config at index creation.
+
+Currently, HybridScan is disabled by default. You can check the configuration page to see how it can be enabled.
```

Comment:
Lets add URL of the configuration page:
".. check the [configuration](02-ug-configuration.md) page ..."